control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/kubevirt.go

```diff
@@ -250,6 +256,40 @@ func reconcileController(controller *appsv1.Deployment, componentImages map[stri
 		}
 	}
 
+	if configoperator.IsExternalInfraKv(hcp) {
+		csiDriverContainerIndex := func() int {
+			for i, container := range controller.Spec.Template.Spec.Containers {
+				if container.Name == "csi-driver" {
+					return i
+				}
+			}
+			return -1
+		}
+		if csiDriverContainerIndex() == -1 {
+			panic(fmt.Errorf("unable to find csi-driver container in %s pod", controllerDeployment.Name))
```

Comment:
please return error - don't panic. it will kill the operator.

Reply:
do we want CPO will continue to run if the `kubevirt-csi-controller` is missing the `csi-driver` container?
i think it should raise a very red flag in this case